lib/execution_engine2/utils/SDKMethodRunner.py

```diff
@@ -294,7 +294,14 @@ def check_job_canceled(self, job_id, ctx):
         Check to see if job is terminated by the user
         :return: job_id, whether or not job is canceled, and whether or not job is finished
         """
-        job_status = self.get_mongo_util().get_job(job_id=job_id).status
+
+        job = self.get_mongo_util().get_job(job_id=job_id)
+
+        if not job:
```

Comment:
Maybe we should just throw the exception in the get_job function if the job is not None so we don't have to check if its None each time.

Reply:
👍

Reply:
Sorry this is a typo, "if the job IS none" we should throw an exception